scripts/beforeInstall.js

```diff
@@ -0,0 +1,20 @@
+module.exports = function(ctx) {
+    var exec = require('child_process').exec;
+    var Q = ctx.requireCordovaModule('q');
+    var deferral = new Q.defer();
+    
+    console.log("Detecting cordova version...");
+    var cordovaV = ctx.opts.cordova.version;
+
+    console.log("Cordova version is " + cordovaV);
+    console.log("Removing file plugin to ensure cordova compatibility...");
+    exec('cordova plugin remove cordova-plugin-file');
+    if (parseFloat(cordovaV) < 6.3) {
+        exec('cordova plugin add cordova-plugin-file-transfer@1.6.3');
```

Comment:
[exec](https://nodejs.org/api/child_process.html#child_process_child_process_exec_command_options_callback) is async function, so we need to pass callback in order to handle exec result correctly.

Reply:
@ruslan-bikkinin, good point! I would prefer to use sync version of this method - https://nodejs.org/api/child_process.html#child_process_child_process_execsync_command_options